### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by Fitzpatrit2**
This adds piece reference and the xml file for the card view where the Pieces on the chess board. It is not finished yet but it should be a good starting point.

### CODE REVIEW

File: PieceReference.kt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.example.groupproject
+
+data class PieceReference(
+    val Rook: String,
+    val Knight: String,
+    val Bishop: String,
+    val King: String,
+    val Queen: String,
+    val Pawn: String,
+)
```
Comment (Torgarr) [inline PieceReference.kt:10]: What is the point of this class?

File: piece_card_view.xml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+                <ImageView
+                    android:id="@+id/Pawn"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    app:layout_constraintWidth_max="20dp"
+                    app:layout_constraintHeight_max="20dp"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toTopOf="parent"
+                    app:layout_constraintBottom_toBottomOf="parent"
+                    />
```
Comment (Torgarr) [inline piece_card_view.xml:18]: All of these image views have the same constraints so they will be overlapping.

**Review by Torgarr — CHANGES_REQUESTED** [review]
1. Both of these files are in the root directory and not in the app directory so the program won't even see these files.
2. where are these files being used at in the app. I do not see any references to these files anywhere

**Review by tylermmonroe — COMMENTED** [review]
I would not upload these files manually. Instead, I would use some kind of Git workflow to properly document changes.

File: PieceReference.kt (hunk 1 above)
Comment (Fitzpatrit2) [reply PieceReference.kt:10]: This is a reference point for the chess pieces for use in the XML files

File: piece_card_view.xml (hunk 2 above)
Comment (Fitzpatrit2) [reply piece_card_view.xml:18]: I am going to change that later, the idea is that each of the views will have different positions when the game is loaded.

**Pull request closed without merging**